src/Microsoft.Kusto.ServiceLayer/DataSource/Kusto/KustoDataSource.cs

```diff
@@ -467,6 +498,41 @@ private IEnumerable<ColumnInfo> GetColumnInfos(string databaseName, string table
             }
         }
 
+        private IEnumerable<ColumnInfo> GetMaterializedViewColumnInfos(string databaseName, string materializedViewName)
+        {
+            ValidationUtils.IsNotNullOrWhitespace(databaseName, nameof(databaseName));
+
+            CancellationTokenSource source = new CancellationTokenSource();
```

Comment:
In other variables you start with var, couldn't these two be started with var too?

`var source = new CancellationTokenSource();`
`var token = source.Token;`